### PR TITLE
Fix PEMA_Orthoimagery_2022

### PR DESCRIPTION
**Pull request by Intrinsica1**
Synchronizes PEMA_Orthoimagery_2022 with JOSM Imagery sources to fix its URL and bounds.

### CODE REVIEW

**Comment by Intrinsica1** [comment]
Note that I used `scripts/convert_individual.py`'s output as reference to modify the original GeoJSON.

File: sources/north-america/us/pa/PEMA_Orthoimagery_2022.geojson [hunk 1]
```diff
@@ -3,14 +3,14 @@
...
...YgLTEuMzI0LDYuNTA2IGggLTEuNTY4IGwgMS43ODcsLTguNzU5IGggMS45NTEgbCAxLjE4MSw0LjI0IDIuODc5LC00LjI0IGggMi4xNjkgeiIvPjxwYXRoIGQ9Im0gNDEuODA2LDM4LjEyOSAtMC4yNTMsLTIuMDM3IGggLTMuMyBsIC0xLjA2LDIuMDM3IGggLTEuNyBsIDQuNjQyLC04Ljc1OSBoIDIuMTY3IGwgMS4xNDUsOC43NTkgeiBtIC0wLjg1NiwtNy4yMTYgLTIuMDM3LDMuODU1IGggMi40ODQgeiIvPjwvZz48ZyBmaWxsPSIjZmZmZmZmIj48cG9seWdvbiBwb2ludHM9IjM4LjM1MywyMC44NzEgMzYuMzg3LDIzLjAzMSAzOC4yOCwyNC43NTggMzUuMTA3LDI0LjQxIDMzLjA5MiwyNi42MjQgMzMuMTE1LDI0LjE5MSAyOS45MDIsMjMuODM3IDMzLjE0NiwyMi43MTggMzMuMjE0LDIwLjMwNiAzNS4xNDUsMjEuOTg1Ii8+PHBvbHlnb24gcG9pbnRzPSIyNy4yMzEsMjEuNzExIDI3LjEzMywyMy41NjkgMjkuNTM5LDIzLjgzMiAyNy4xMTksMjQuNzM4IDI2Ljk3NSwyNi41MjIgMjUuNTQzLDI1LjI1MiAyMy4xMjMsMjYuMDAxIDI0LjYwNCwyNC4zNzIgMjMuMjQ2LDIzLjA2MyAyNS42NDEsMjMuMzQ2Ii8+PHBvbHlnb24gcG9pbnRzPSIyMi44MDMsMjYuMTMxIDIxLjQ2NSwyNy40MDEgMjIuMzQsMjguNTI3IDIwLjU2LDI4LjE1IDE5LjE2MywyOS4zNzQgMTkuNDAxLDI3LjkyOCAxNy42MTksMjcuNTI0IDE5LjUxOSwyNy4wMzEgMTkuODI5LDI1LjU0OSAyMC44MDUsMjYuNjMxIi8+PHBhdGggZD0ibSAzOC44NDksMjQuOTA2IGEgOC45MjQsOC45MjQgMCAwIDEgNC45NjgsMy4zNjggYyAxLjQzNCwyLjIyMiAtMC4xLDUuMDcyIC0wLjEsNS4wNzIgYSA2LjU3OCw2LjU3OCAwIDAgMCAtMS4zMTksLTUuNjI5IDkuNzMyLDkuNzMyIDAgMCAwIC0zLjU0OSwtMi44MTEgeiIvPjwvZz48L3N2Zz4=",
```
Comment (1ec5) [inline sources/north-america/us/pa/PEMA_Orthoimagery_2022.geojson]: Normally we check images directly into the repository as ordinary files. It looks like scripts/unembed-icons.py will take care of this for you.

File: sources/north-america/us/pa/PEMA_Orthoimagery_2022.geojson [hunk 2]
```diff
@@ -3,14 +3,14 @@
...
-            "text": "Pennsylvania Emergency Management Agency",
+            "url": "https://www.pasda.psu.edu/uci/DataSummary.aspx?dataset=5177",
+            "text": "PASDA",
             "required": false
         },
-        "name": "PEMA Orthoimagery - Western (2022)",
-        "icon": "https://www.pema.pa.gov/PublishingImages/logo.svg",
-        "url": "https://imagery.pasda.psu.edu/arcgis/services/pasda/PEMAImagery2022/MapServer/WmsServer?request=getmap&version=1.3.0&service=wms&layers=2&styles=&format=image/jpeg&crs={proj}&width={width}&height={height}&bbox={bbox}",
-        "max_zoom": 18,
+        "name": "PEMA Orthoimagery [2022]",
```
Comment (1ec5) [inline sources/north-america/us/pa/PEMA_Orthoimagery_2022.geojson]: I think the previous name would be more accurate. Consistent with this layer’s official title, “PEMA Orthoimagery Western PA”, it covers only a slice of western Pennsylvania (including Pittsburgh). Perhaps the JOSM index could use that name instead?

For reference, a similarly named layer covers the whole state:

https://github.com/osmlab/editor-layer-index/blob/4388011bbba74926d3622a6c6355a05e73ad8e00/sources/north-america/us/pa/PEMA_Orthoimagery.geojson#L10

File: sources/north-america/us/pa/PEMA_Orthoimagery_2022.geojson [hunk 3]
```diff
@@ -3,14 +3,14 @@
     "properties": {
         "id": "PEMA_Orthoimagery_2022",
         "attribution": {
-            "url": "https://www.pema.pa.gov/911-Program/Partners/NG911-GIS/Aerial-Imagery/Pages/default.aspx",
-            "text": "Pennsylvania Emergency Management Agency",
+            "url": "https://www.pasda.psu.edu/uci/DataSummary.aspx?dataset=5177",
+            "text": "PASDA",
```
Comment (1ec5) [inline sources/north-america/us/pa/PEMA_Orthoimagery_2022.geojson]: PEMA is the author and PASDA is the host, right? The previous text would be more accurate, although it doesn’t matter that much since attribution isn’t required.

File: sources/north-america/us/pa/PEMA_Orthoimagery_2022.geojson (hunk 1 above)
Comment (Intrinsica1) [reply sources/north-america/us/pa/PEMA_Orthoimagery_2022.geojson]: the script unfortunately didn't seem to work/do anything

**Comment by Intrinsica1** [comment]
really appreciate the reviews, thanks :)

**Comment by Intrinsica1** [comment]
Just to inform that I believe it is ready to merge. Also to reiterate that this layer is currently in a nonfunctional state in the master branch.

**Comment by 1ec5** [comment]
The build is failing because the icon hasn’t been deployed yet, but obviously that’s not your fault:

> ERROR:root:sources/north-america/us/pa/PEMA_Orthoimagery_2022.geojson: icon url https://osmlab.github.io/editor-layer-index/sources/north-america/us/pa/PEMA.svg is not reachable: HTTP code: 404

I see that most other sources just link to the URL directly. We could link to https://www.pema.pa.gov/PublishingImages/logo.svg , but I think what you have should be fine once we merge.

**Comment by Intrinsica1** [comment]
The reason I didn't link the original SVG is that I modified it to render properly in JOSM (both still visually the same). Though this is the fault of JOSM, and not the SVG.

**Comment by 1ec5** [comment]
Makes sense. I also noticed that, for a couple layers, someone had also converted the SVG to PNG and checked in the PNG, maybe for the same reason.

**Comment by cicku** [comment]
Any reason you need to upload the icon to the repo, also @1ec5 did you notify me for any PEMA changes?

**Comment by 1ec5** [comment]
> Any reason you need to upload the icon to the repo

@Intrinsica1 said above that the SVG from the site didn’t render well when ELI is loaded in JOSM, so they checked in a simplified version. The JOSM rendering issue is not unexpected, since every SVG renderer has its own quirks; another ELI consumer could well have the same issue.

If the checked-in icon is an issue, we could just omit the icon; it’s nothing more than a nicety.

**Comment by cicku** [comment]
Although I personally wish JOSM team the best on their bugfix journey, this issue along with another one #2337 requires me to emphasize again that **ELI is not the correct place to fix/discuss whatever problem that JOSM possesses.**

There will always be bugs/quirks in JOSM since it is technically a software, and no software is perfect. JOSM can have all kinds of problems but it is not our objective to find a workaround, we are only a layer index providing the information as-is, and our consumers do not just include JOSM.

**Comment by 1ec5** [comment]
I get your point and agree in general, but SVG is a very fragile format. If JOSM is having problems with this particular SVG, it’s entirely possible that iD would when viewed in some less popular browser. But it’s hard to say without more information. If @Intrinsica1 could provide a link to the bug report in JOSM’s Trac instance, we could try to work toward a resolution there. Or we could shrug and delete the icon from this repository. It’s quite a trivial matter – I don’t even think this icon would be all that legible in the corner of iD against the average PEMA imagery.

**Comment by simonpoole** [comment]
May I point out that this is supposed to an editor neutral index of imagery. 

While PNG and JPEG are nearly universally supported on computing devices, the same can not be said of SVG, for example Android has no native SVG support, which just effects the small number of 4 billion devices. There are 3rd party libraries to provide such support, but, seriously, I don't think it is a good policy to expect SVG support to be added just for a couple of manufacturer icons.

tl;dr keep the icons in the repo to PNG and JPEG.

**Comment by cicku** [comment]
> May I point out that this is supposed to an editor neutral index of imagery.

Of course I agree.

> While PNG and JPEG are nearly universally supported on computing devices, the same can not be said of SVG, for example Android has no native SVG support, which just effects the small number of 4 billion devices. There are 3rd party libraries to provide such support, but, seriously, I don't think it is a good policy to expect SVG support to be added just for a couple of manufacturer icons.

Then the CONTRIBUTING.md needs to be updated.
